game-core/src/test/java/games/strategy/triplea/UnitUtilsTest.java

```diff
@@ -125,5 +128,72 @@ void bombingDamageIsNotTransferredIfDamageAllowedIs0() {
           newUnits.get(0).getUnitDamage(),
           is(0));
     }
+
+    @Test
+    void unloadedUnitsAreTransferred() {
```

Comment:
What is meant by transferred? The part that is unclear, is I'm not sure where they are transferred. Suggestion: use a `@DisplayName` to give more description about this meaning.